src/realtime.js

```diff
@@ -143,10 +143,18 @@
         });
     },
 
+    ping: function () {
+      var self = this;
+
+      this.socket.emit('ping', { timestamp: new Date().getTime() } , function (name, data) {
```

Comment:
`new Date().getTime()` is fine but you could have used `+(new Date())` instead as we are already using this format in the code